src/sections/Home/So-Special-Section/so-special-style.js

```diff
@@ -7,6 +7,8 @@ const SoSpecialWrapper = styled.div`
     align-items:center;
     padding:0rem 3rem 2rem 3rem;
     overflow:hidden;
+    font-family:"Qanelas Soft";
+    font-weight:600;
```

Comment:
Should be updated, while updating the above.

Reply:
Thanks for the review . I 'll make the required changes !